Dockerfile

```diff
@@ -34,9 +34,13 @@ FROM alpine:3.14
 
 WORKDIR /app
 
+ENV MODULES_ASSETS=modules_assets
```

Comment:
yes, right now it is not used, but we have modules with assets and we need a common way to store the assets

Reply:
assets are just static files. Some modules require additional files for configuration, testing, etc.

Reply:
@FrimIdan, MODULES_ASSETS is a directory, with a well known structure, where each modules can put its private assets.
For example the trace analyzer can have a bunch of files which are dictionaries of weak passwords.

Reply:
ok, but where/how the ENV var is used?

Reply:
added a function that uses this env, previously each module that needed assets was using this const directly with `os.Getenv`

Reply:
It's a bit strange for me to configure the env in the dockerfile and not in the k8s deployment spec.
Also all assets can be mounted into the deplyoment instead of a copy during build time.
This was done to support local development and testing?

Reply:
Of course, you are right, the env can and should be configured during the k8s deployment.
But in case of local usage, it's good to have a working self contained docker image, with minimal working defaults.

Each modules, would be responsible to look into MODULES_ASSETS/module_name (/opt/apiclarity/assets/traceanalyzer, for example), in order to find what it needs.

So, at the end, in the docker image you have a set of default simple assets for each modules, if needed.
And then, if one want to have its own set of assets, he changes the MODULES_ASSETS location and put his own assets, during deployment.